### DISCUSSION TITLE
Change page title and logo

### DISCUSSION THREAD

**Discussion by kerim371** [Kitware/trame#607] [Q&A] [answered]
Hi!

Is it possible to change page title and logo?
![image](https://github.com/user-attachments/assets/84e2f7ad-89e5-4632-aaa3-caf68647bf31)

**Comment by jourdain** [comment]
Yes that is covered in the readme of the repo along within [that example](https://github.com/Kitware/trame/blob/master/examples/core_features/reserved_state.py)

**Reply by kerim371** [reply]
Thank you! That answers the question.

One note: according to [the documentation](https://trame.readthedocs.io/en/latest/#reserved-state-entries) favicon `needs to be an image encoded url`.
If I have it as an `.ico` image stored on the laptop disk how can I make it use?

**Reply by jourdain** [reply, marked as answer]
Another example [here](https://github.com/Kitware/trame/blob/master/examples/validation/core/12_reserved_state.py)

**Reply by kerim371** [reply]
Thank you very much! That helped
